src/profile/Index.vue

```diff
@@ -81,6 +101,8 @@ export default {
   }),
   computed: {
     hasUnverifiedEmails: vm => vm.alternates.some(m => ! m.verified),
+    additionalKeys: vm => vm.mfa.keys.data,
```

Comment:
What is the data structure here? I expected `keys` to be an array of webauthn keys, but is that actually down in `data`?